src/thunderbots/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     set_property(CACHE CMAKE_BUILD_TYPE PROPERTY VALUE "Release")
 endif()
 
+# For Visual Studio Code Intellisense
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
:+1: this works for vim too! Might want to change comment to something like `# For external editor code completion and hinting (Visual Studio Code, Vim, etc.)`

Reply:
Ah, didn't know Vim (well, at least YCM) uses this .json too. Will update.